src/metabase/driver/mongo/util.clj

```diff
@@ -1,6 +1,7 @@
 (ns metabase.driver.mongo.util
   "`*mongo-connection*`, `with-mongo-connection`, and other functions shared between several Mongo driver namespaces."
   (:require [clojure.tools.logging :as log]
+            [metabase.util.ssh :as ssh]
```

Comment:
That was auto-placed by clj-refactor cleanup namespace.  which just sorts them alphabetically and consolidates them where possible. It's conventient because it automatically keeps all `ns` delarations consistent.